roles/remove-unattended-upgrades/tasks/main.yaml

```diff
@@ -0,0 +1,15 @@
+# apt supports lock_timeout from 2.12 (impish onwards) until then manually
+# check for the lock.
+- name: Wait for apt lock
+  command: /usr/bin/lsof /var/lib/dpkg/lock-frontend
+  retries: 30
+  delay: 10
+  register: result
+  until: result.rc == 1
+  failed_when:
+    - result.rc == 0
+- name: Remove unattended-upgrades
```

Comment:
I wonder if we should add a comment about why we are doing this and inform anyone looking at this repository for inspiration to not do this at home, similar to:
https://github.com/openstack-charmers/zaza/blob/f3600758b976ff0aed2b70acaa383eeaaadcc69b/.github/workflows/tox.yaml#L45